point/ㄹ_바라다.yaml

```diff
@@ -0,0 +1,29 @@
+name: (으)ㄹ 바라다
+definitions:
+  - slug: hope
+    name: Hope
+    english_alternatives: to hope, to wish
+    meaning: Used to express the speaker's hope or wish for something to happen.
+    examples:
+      - sentence: 여러분의 성공을 기원하며 행복한 미래를 <f>가지시길 바랍니다</f>.
+        type: simple
+        translated: I hope for your success and wish you a happy future.
+        audio_url:
+      - sentence: 모두가 안전하게 <f>돌아오시길 바랍니다</f>.
+        type: simple
+        translated: I hope everyone returns safely.
+        audio_url:
+      - sentence: 모두의 <f>행복을 바랍니다</f>.
+        type: simple
+        translated: I hope for everyone's happiness.
+        audio_url:
+metadata:
+  type: composite
+details: |-
+  # Usage with verbs {#usage-with-verbs}
+
+  When used with action or descriptive verbs it's often used in the form <f>-기 바라다</f>.
```

Comment:
```suggestion
  The structure starts with a :grammar[noun_을_를]{object} which can sometimes be
  omitted if implied. This is why you may see the phrase <f>"-기(를) 바라다"</f> in
  grammar books. Since :grammar[ㄹ_바라다]{hope} requires a noun as its object,
  verbs must first be nominalized using :grammar[verb_기]{verb-to-noun}.```